rust/hyperlane-base/src/settings/mod.rs

```diff
@@ -128,10 +129,14 @@ static KMS_CLIENT: OnceCell<KmsClient> = OnceCell::new();
 pub struct Settings {
     /// Configuration for contracts on each chain
     pub chains: HashMap<String, ChainSetup>,
+    /// Default signer configuration for chains which do not define their own.
+    /// This value is intentionally private as it will get consumed by
+    /// `post_deserialize`.
+    defaultsigner: Option<SignerConf>,
```

Comment:
Should this be specific to relayer settings?

Reply:
See other comment